src/main/game/gameboard/areaEffect/AreaEffectPower.java

```diff
@@ -0,0 +1,22 @@
+package game.gameboard.areaEffect;
+
+/**
+ * Created by khariollivierre on 3/13/17.
+ */
+public class AreaEffectPower {
+    private int power;
+
+    public AreaEffectPower(AreaEffectEnum effect){
+        switch (effect){
+            case POISON: power = 5;     // Do this.power damage to units on tile
```

Comment:
these probably deserve subclasses since death is a special case. doing -1 like this is mixed-instance cohesion. Also, don't forget to put a break after each case statement